lib/neuron/config.ex

```diff
@@ -26,18 +42,30 @@ defmodule Neuron.Config do
 
       iex>Neuron.Config.set(url: "http://example.com/graph"); Neuron.Config.get(:url)
       "http://example.com/graph"
+
+      iex>Neuron.Config.set(headers: [name: "value"]); Neuron.Config.get(:headers)
```

Comment:
You can actually do this in two lines, no need to add the `;` delimiter. It would read better.